src/components/RecipeCard/RecipeCard.js

```diff
@@ -87,6 +86,13 @@ const Section = styled.section`
     'items';
   grid-gap: 5px;
   margin-bottom: 8px;
+  max-height: 45vh;
+  overflow: auto;
+  ${({ showDetails }) =>
+    showDetails &&
+    `
+          max-height: none;
+        `};
```

Comment:
There is a lot of superfluous white space in that template string. That makes reading your code a little bit harder. :)
```suggestion
  ${({ showDetails }) =>
    showDetails && `max-height: none;`
};
```

Reply:
Yes, I can see that. Thanks for the suggestion! 😸